frontend/src/pages/UI/Breadcrumb.jsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
```

Comment:
Maybe move the `UI` directory outside `pages`?  Maybe just call it `components`?

Reply:
That would be my preference too - I will move it.